apps/coscrad-frontend-e2e/src/e2e/resources/terms/term.index-list-pagination.e2e.cy.ts

```diff
@@ -0,0 +1,252 @@
+                cy.getLoading().should('not.exist');
+
+                cy.getByDataAttribute(`index_search_bar`).click();
+
+                cy.getByDataAttribute(`index_search_bar`).type('You can find me #1.');
+
+                cy.getLoading().should('not.exist');
+
+                cy.contains('Page: 1/1');
+
+                cy.contains(`Filtered Records: 1`);
+
+                cy.contains(`Showing Records: 1-1/1`);
+            });
+        });
+
+        describe(`when the filters find several pages of results`, () => {
+            beforeEach(() => {
+                cy.seedDatabase('term__VIEWS', allTerms);
+
+                cy.visit(termIndexRoute);
+
+                cy.get('#mui-component-select-pageSize').click();
+
+                cy.get('#mui-component-select-pageSize')
+                    .get(`[data-value="${targetPageSize}"`)
+                    .click();
+            });
+
+            it(`should lazily load each page properly`, () => {
+                cy.getLoading().should('not.exist');
+
+                cy.getByDataAttribute(`index_search_bar`).click();
+
+                cy.getByDataAttribute(`index_search_bar`).type(searchTerms);
+
+                cy.getLoading().should('not.exist');
+
+                cy.contains(`Page: 1/3`);
+                /**
+                 * Alphabetically
+                 * 0
+                 * 1
+                 * 10
+                 * 11
+                 * 12
+                 */
+
+                cy.contains(`You can find me #1.`);
+
+                cy.contains(`You can find me #13.`).should('not.exist');
+
+                cy.getByDataAttribute('ArrowForwardIosIcon').click();
+
+                cy.contains(`Page: 2/3`);
+                /**
+                 * Alphabetically
+                 * 13
+                 * 14
+                 * 2
+                 * 3
+                 * 4
+                 */
+
+                cy.getLoading().should('not.exist');
+
+                cy.contains(`You can find me #1.`).should('not.exist');
+
+                cy.contains(`You can find me #3.`);
+
+                cy.getByDataAttribute('ArrowForwardIosIcon').click();
+
+                cy.contains(`Page: 3/3`);
+                /**
+                 * Alphabetically
+                 * 5
+                 * 6
+                 * 7
+                 * 8
+                 * 9
+                 */
+
+                cy.getLoading().should('not.exist');
+
+                cy.contains(`You can find me #1.`).should('not.exist');
+
+                cy.contains(`You can find me #8.`);
+
+                cy.getByDataAttribute('ArrowForwardIosIcon').click();
+
+                // here we ensure that the page carousel "wraps around"
+                cy.contains(`Page: 1/3`);
+
+                cy.getLoading().should('not.exist');
+
+                cy.contains(`You can find me #9.`).should('not.exist');
+
+                // TODO Change this to the new icon
```

Comment:
Remove this comment. The new icon only exists for the back arrow due to some historical quirk.